### ISSUE TITLE
TLS support?

### ISSUE THREAD

**Issue by mzhang28** [worblehat/FredIRC#4]
Doesn't look like this exists as an option... is anyone not using TLS these days?

**Comment by worblehat** [comment]
Right. Unfortunately TLS support is missing. It's an important point on the "wishlist" but I haven't worked on FredIRC for years.

Python asyncio supports SSL via [create_connection()](https://github.com/worblehat/FredIRC/blob/master/fredirc/client.py#L403). To be honest, I haven't used this API in a while and I am not sure which other implications this has for the rest of the code.

I'll try to find some time to look into this...but can't promise anything right now.
